spec/FilesController.spec.js

```diff
@@ -218,4 +218,99 @@ describe('FilesController', () => {
     expect(gridFSAdapter.validateFilename(fileName)).not.toBe(null);
     done();
   });
+
+  it('should return valid filename or url from createFile response when provided', async () => {
+    const config = Config.get(Parse.applicationId);
+
+    // Test case 1: adapter returns new filename and url
```

Comment:
Could you please split this into separate tests, each one with a distinct title to describe what it's testing.

Reply:
You also may want to take a look at the failing tests in the CI

Reply:
Done, sorry about the delay